README.md

```diff
@@ -11,45 +17,45 @@ Note that ruby 1.8 support was dropped in ruby-oci8 2.2.0.
 Use ruby-oci8 2.1.8 for ruby 1.8.
 
 What's new
-==========
+----------
 
 See {file:NEWS}.
 
 Sample one-liner
-================
+----------------
 
 Connect to scott/tiger, select `emp` and print as CSV format.
 
     ruby -r oci8 -e "OCI8.new('scott', 'tiger').exec('select * from emp') do |r| puts r.join(','); end"
 
 Homepage
-========
+--------
 
 * http://www.rubydoc.info/github/kubo/ruby-oci8
 
 Installation
-============
+------------
```

Comment:
@Gaurav2728 
I agree that `##` is more suitable for change.
However I favour `------` because it emphasizes headings than `##` when I look it in a text editor.